src/components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import TextArea from "./TextArea";
+
+export default {
+  title: "Component/TextArea",
```

Comment:
g ush pake component, since semuanya component wkwk unless there is a new logical class

Reply:
Ah iy gw mw buat logical class. Kyk mgkn Inputs, Data Display, etc